dspace/bin/dspace-info.pl

```diff
@@ -53,17 +53,17 @@
 
 # count DSpace objects ######################
 
-my $bitstream_count     = CountRows( "bitstream"     );
-my $bundle_count        = CountRows( "bundle"        );
-my $collection_count    = CountRows( "collection"    );
-my $community_count     = CountRows( "community"     );
-my $dcvalue_count       = CountRows( "dcvalue"       );
-my $eperson_count       = CountRows( "eperson"       );
-my $item_count          = CountRows( "item"          );
-my $handle_count        = CountRows( "handle"        );
-my $group_count         = CountRows( "epersongroup"  );
-my $workflowitem_count  = CountRows( "workflowitem"  );
-my $workspaceitem_count = CountRows( "workspaceitem" );
+my $bitstream_count     = CountRows( "bitstream"       );
+my $bundle_count        = CountRows( "bundle"          );
+my $collection_count    = CountRows( "collection"      );
+my $community_count     = CountRows( "community"       );
+my $dcvalue_count       = CountRows( "dcvalue"         );
```

Comment:
this script is obviously out dated, should we remove it completely? this can be done in a different PR obviously but we should have a ticket to remove it or fix. I would prefer removal as it also imply the need to have perl

Reply:
Agreed.  I'll push up a commit to remove this in #3161, since I'm removing a lot of other unnecessary files in that PR.